lib/l10n/intl_en_US.xml

```diff
@@ -3097,4 +3097,28 @@
     name="replyDraftsLabel"
     description="Text label for Drafts destination."
     >Drafts</string>
+  <string
+    name="demoTwoPaneDualScreenLabel"
+    description="Option title for TwoPane demo in dual screen mode."
+    >Dual Screen</string>
```

Comment:
Yes. I will go through all the text and make sure dual-screen is replaced with foldable. I started working on this at a time where our focus was dual-screen. We have since switched our focus to be on all foldable devices.